video/out/vo_sixel.c

```diff
@@ -40,6 +39,8 @@
 
 #define TERMINAL_FALLBACK_DEFAULT_WIDTH 80
 #define TERMINAL_FALLBACK_DEFAULT_HEIGHT 25
+#define IMG_FALLBACK_DEFAULT_WIDTH 320
```

Comment:
These should be renamed to TERMINAL_FALLBACK_PX_WIDTH (/HEIGHT) because it's no longer the image size.

Also TERMINAL_FALLBACK_DEFAULT_WIDTH  (and HEIGHT) could be shortened/changed to TERMINAL_FALLBACK_ROWS (/COLS).

Reply:
Have renamed them accordingly.